### PR TITLE
Polish for .NET Web guides

### PR DESCRIPTION
**Pull request by laura-rodriguez**
## Description:
- **What's changed?** 
* Update all .NET examples and make sure all code samples start with ```csharp.
* Expand getuserinfo.md to show examples of accessing claims in both controllers and views.

- **Is this PR related to a Monolith release?** 
No

### CODE REVIEW

File: packages/@okta/vuepress-site/guides/sign-into-web-app/get-user-info/aspnet/getuserinfo.md [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
     ...
     return View();
 }
-```
\ No newline at end of file
+```
+
+Or in your views:
+
+```csharp
```
Comment (OkGoDoIt) [inline packages/@okta/vuepress-site/guides/sign-into-web-app/get-user-info/aspnet/getuserinfo.md]: This should be ` ```cshtml`

File: packages/@okta/vuepress-site/guides/sign-into-web-app/get-user-info/aspnetcore/getuserinfo.md [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
 {
     var userClaims = HttpContext.User.Claims;
     ...
     return View();
 }
+```
+
+Or in your views:
+
+```csharp
```
Comment (OkGoDoIt) [inline packages/@okta/vuepress-site/guides/sign-into-web-app/get-user-info/aspnetcore/getuserinfo.md]: This should be ` ```cshtml`

File: packages/@okta/vuepress-site/guides/sign-into-web-app/redirect-to-sign-in/aspnet/login-redirect.md [hunk 3]
```diff
@@ -2,7 +2,7 @@ When accessing protected routes, ASP.NET redirects the user to an Okta sign-in p
 
 You can also give the user a **Sign In** button or link. Open your `_Layout.cshtml` file and add the following code:
 
-```
+```csharp
```
Comment (OkGoDoIt) [inline packages/@okta/vuepress-site/guides/sign-into-web-app/redirect-to-sign-in/aspnet/login-redirect.md]: This should be ` ```cshtml`

File: packages/@okta/vuepress-site/guides/sign-into-web-app/redirect-to-sign-in/aspnetcore/login-redirect.md [hunk 4]
```diff
@@ -2,7 +2,7 @@ When accessing protected resources, ASP.NET Core redirects the user to an Okta s
 
 You can also give the user a **Sign In** button or link. Open your `_Layout.cshtml` file and add code the following code:
 
-```
+```csharp
```
Comment (OkGoDoIt) [inline packages/@okta/vuepress-site/guides/sign-into-web-app/redirect-to-sign-in/aspnetcore/login-redirect.md]: This should be ` ```cshtml`

**Comment by nbarbettini** [comment]
Thanks @laura-rodriguez and @OkGoDoIt!

**Pull request merged**